src/pages/VoiceRoomPage/VoiceRoomListPage.styled.ts

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+
+import yellow from "@/assets/VoiceRoom/yellow_gradient.svg";
+import purple from "@/assets/VoiceRoom/purple_gradient.svg";
+
+export const DropdownDiv = styled.div`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	visibility: hidden;
+	z-index: 1;
+	color: var(--Foundation-Gray-white, #fff);
+	margin: 0rem 0rem 0.5rem 1rem;
+	padding: 1rem;
+	width: 7.5rem;
+
+	/* text/Regular 14pt */
+	font-family: Freesentation;
+	font-size: 0.875rem;
+	font-style: normal;
+	font-weight: 400;
+	line-height: 140%; /* 19.6px */
+	letter-spacing: 0.035rem;
+	border-radius: 0.75rem;
+	background: var(--GRAY-800, #222226);
+`;
+
+export const ActiveP = styled.p`
+	font-family: "Freesentation SB";
+	font-size: 1.125rem;
+`;
+export const VRTitleDiv = styled.div`
+	font-family: "Freesentation SB";
+	font-size: 1.25rem;
+	color: black;
+	margin: 1rem;
+`;
+export const InfoDiv = styled.div`
+	display: flex;
+	flex-direction: column;
+	&:hover ${DropdownDiv} {
+		visibility: visible;
+	}
+`;
+export const BGdiv = styled.div`
+	display: flex;
+	flex-direction: column;
+	background-image: url(${yellow});
+
```

Comment:
```suggestion
    background-repeat: no-repeat;
    background-size: contain;
```

이 부분 추가하면 아래 처럼 뜨긴 하는데,
항상 width 100% height 100%를 의도하는 건가요?
![image](https://github.com/user-attachments/assets/199720ce-c3e4-4ff6-8b9e-fe2cabc39684)

---
일반적인 gradation 배경만을 원하는 거라면, image 배경 대신 css gradation 기능으로 직접 만드는 것이 크기 변환에 더 좋을 수 도 있을 것 같아요

https://developer.mozilla.org/ko/docs/Web/CSS/gradient/linear-gradient

Reply:
![image](https://github.com/user-attachments/assets/8eca0065-e54c-4772-8a79-c84c16f7c706)
대신 그렇게 되면 width가 끝까지 안 차더라구요
궁극적으로는 css gradation으로 해야할거 같아요